fintech-examples/fintech-impl/src/main/java/de/adorsys/opba/fintech/impl/config/FeignConfig.java

```diff
@@ -46,7 +46,7 @@ public AisErrorDecoder aisErrorDecoder() {
     }
 
     @Bean
-    @Profile("!no-signature-filter")
+    @Profile("!no-signature-filter | !security-bypass")
```

Comment:
Extract `!no-signature-filter | !security-bypass` to global constant

Reply:
moved